### DISCUSSION TITLE
How to change gray color on the mesh?

### DISCUSSION THREAD

**Discussion by pswpswpsw** [pyvista/pyvista#6298] [Q&A] [answered]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Write a short description telling me what you are doing, what you expect to happen, and what is currently happening.

```python
tmp = pv.read(folder_name + '_post/'+f'{NORM}_mean.vtm')
plotter = pv.Plotter(off_screen=True,shape=(2, 3),window_size=[1200, 800])

plotter.subplot(0,0)
plotter.add_mesh(tmp['internal'],scalars='rho mean', clim=[9e-4,0.01],show_scalar_bar=False)
plotter.camera_position = 'xy'
_ = plotter.add_scalar_bar(
        'rho',
        interactive=False,
        vertical=False,
        title_font_size=15,
        label_font_size=15,
        outline=False,
        fmt='%.e')
```

### Description

I am just visualizing a 2D mesh plot. My color map is RdBu so I can see the variations around zero. However, the background always has a gray color. How to change such gray color to white and only keep the edge? 

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Mon Jun 24 13:23:56 2024 EDT

                OS : Linux
            CPU(s) : 64
           Machine : x86_64
      Architecture : 64bit
               RAM : 251.5 GiB
       Environment : Python
       File system : ext4
        GPU Vendor : NVIDIA Corporation
      GPU Renderer : NVIDIA GeForce RTX 4090/PCIe/SSE2
       GPU Version : 4.5.0 NVIDIA 535.161.07
  MathText Support : False

  Python 3.10.11 (main, Apr 20 2023, 19:02:41) [GCC 11.2.0]

           pyvista : 0.43.1
               vtk : 9.0.3
             numpy : 1.23.5
        matplotlib : 3.6.2
            scooby : 0.9.2
             pooch : 1.4.0
            pillow : 9.4.0
             PyQt5 : 5.15.7
           IPython : 8.8.0
        ipywidgets : 8.0.4
             scipy : 1.10.0
              tqdm : 4.65.0
        jupyterlab : 3.3.2
      nest_asyncio : 1.5.6

  Intel(R) oneAPI Math Kernel Library Version 2021.4-Product Build 20210904
  for Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

### Screenshots

![image](https://github.com/pyvista/pyvista/assets/7966776/0798ec50-526a-4268-b358-41efc413b334)

**Comment by user27182** [comment, marked as answer]
You can try turning off the lighting for the mesh, e.g.
``` python
add_mesh(mesh, lighting=False)
```
For example, a white sphere looks gray with lighting enabled (default):
``` python
import pyvista as pv
pv.Sphere().plot(color='white')
```
<img src="https://github.com/pyvista/pyvista/assets/89109579/5d722159-d7ae-42d3-a4d6-5874b008c0ee" width="300">

But it will completely disappear into the white background with `lighting=False`
```python
import pyvista as pv
pv.Sphere().plot(color='white', lighting=False)
```
<img width="300" alt="image" src="https://github.com/pyvista/pyvista/assets/89109579/5209287e-adf7-4f67-8d8f-ded7a6e34a8c">

**Comment by pswpswpsw** [comment]
Thanks! It works!
